### ISSUE TITLE
Add template_searchpath support

### ISSUE THREAD

**Issue by tiger24l7** [bryzgaloff/airflow-clickhouse-plugin#49]
Hi! 
The problem is that `ClickhouseOperator` doesn't support `template_searchpath` dag option which allows to pass filename instead of full query

Instead of:
```python
ClickHouseOperator(
    task_id='task_1',
    clickhouse_conn_id='clickhouse_default',
    sql='SELECT * from db.some_table;' # <-- raw SQL query
)
```

We can write:
```python
ClickHouseOperator(
    task_id='task_1',
    clickhouse_conn_id='clickhouse_default',
    sql='select_query.sql' # <-- filename of jinja template which is located in template_searchpath location
)
```

[One more example how it works
](https://docs.astronomer.io/learn/dag-best-practices#treat-your-dag-file-like-a-config-file)

**Comment by tiger24l7** [comment]
Looks like it's already done https://github.com/whisklabs/airflow-clickhouse-plugin/pull/41

**Comment by bryzgaloff** [comment]
Hi @tiger24l7, thank you for you feedback. Yes, there is an open PR, feel free to contribute :)

**Comment by tiger24l7** [comment]
My python skills are poor
But maybe, I'll try to implement this later
Would be nice if someone handle this now, if possible :)

**Comment by bryzgaloff** [comment]
Contribution is welcome, thank you! I will review you PR and let you know how you may enhance the code.

**Comment by bryzgaloff** [comment]
Implemented in #62
